tests/integration/pilot/traffic_shifting_test.go

```diff
@@ -66,57 +68,103 @@ type VirtualServiceConfig struct {
 	Weight2   int32
 }
 
+// Test case for regular echo instances
 func TestTrafficShifting(t *testing.T) {
+	trafficShifting(t, []string{}) // pass empty list to indicate no VM
+}
+
+// Test case for VM traffic shifting. Client a will sent requests to 3 VMs
+// VM image version is default to ubuntu:bionic
```

Comment:
nit - just say version is default & drop the value so that when we update the default we don't have to change code to reflect it

Reply:
I will update! Thanks for the comment!